recipe/meta.yaml

```diff
@@ -9,44 +8,83 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
+    sha256: {{ sha256 }}
+    folder: go
+  - url: https://storage.googleapis.com/golang/go1.4-bootstrap-20170531.tar.gz
+    sha256: 49f806f66762077861b7de7081f586995940772d29d4c45068c134441a743fa2
+    folder: go-bootstrap
 
 build:
-  number: 0
-  # It's relocatable. $GOROOT FTW!
-  binary_relocation: False
```

Comment:
Why has this been removed? Has this been dropped deliberately?

Reply:
I think we should also add `detect_binary_files_with_prefix: False`

Reply:
It was removed deliberately for the Linux/OSX builds. However it can be added back for Windows to speed up the build time.

On Linux and OSX we can let conda do its traditional binary relocation, i.e. replacing the long 255 character string with the installed prefix. To accomplish this I set the GOROOT_FINAL in the build.sh scripts just like it was done in debian. Here is the snippet of the code:
```bash
# Exports copied from
#  https://salsa.debian.org/go-team/compiler/golang/blob/golang-1.10/debian/rules#L9
export GOROOT=$SRC_DIR/go
export GOROOT_FINAL=${PREFIX}/go
export GOCACHE=off
```
I am not a conda developer, but I made this decision under the assumption that it was better to let it handle the relocation whenever possible.

On Windows the "GOROOT_FINAL" environment has no effect, i.e. Go either needs to be installed to the original $PREFIX, or we need to set GOROOT during environment activation.  I did not find out about this issue until I started working on the windows build a long time later.

If you prefer to restore symmetry to build scripts, let me know and I can adjust the build scripts so that the Linux and OSX builds don't use GOROOT_FINAL either.

Reply:
Point taken, I will work on getting the relocation back to normal.